lib/segment/src/id_tracker/simple_id_tracker.rs

```diff
@@ -154,9 +154,9 @@ impl SimpleIdTracker {
         );
         for (internal, external) in internal_to_external.iter_mut().enumerate() {
             if deleted[internal] {
-                log::debug!("Adding synthetic version 0 to a deleted point, external id: {external}, internal id: {internal}");
                 // Add synthetic version 0
                 internal_to_version.push(0);
+                log::debug!("Added synthetic version 0 to a deleted point, external id: {external}, internal id: {internal}");
```

Comment:
Maybe its better to remove this one, I can imagine it to be noisy.

We don't want 100k log lines if we have 1 million points with 10% deleted.

```suggestion
```

Reply:
Good point